backend/src/main/java/com/tcs/dhv/service/TrackingService.java

```diff
@@ -0,0 +1,55 @@
+package com.tcs.dhv.service;
+
+import com.tcs.dhv.domain.dto.TrackingDto;
+import com.tcs.dhv.domain.entity.Parcel;
+import com.tcs.dhv.repository.ParcelRepository;
+import jakarta.persistence.EntityNotFoundException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+
+import java.util.Optional;
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class TrackingService {
+
+    private final ParcelRepository parcelRepository;
+
+    public TrackingDto getTrackingDetails(
+            String trackingCode
+    ){
```

Comment:
```suggestion
    public TrackingDto getTrackingDetails(String trackingCode){
```
no need to have it on it's own line